domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/response/HowMuchOwed.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.cmc.domain.models.response;
+
+import org.hibernate.validator.constraints.NotBlank;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+
+import javax.validation.constraints.Size;
+
+
+public class HowMuchOwed {
+
+    @NotBlank
+    private final BigDecimal amount;
+
+    @NotBlank
+    @Size(max = 255, message = "must be at most {max} characters")
+    private final String text;
```

Comment:
I'd think about better name for that field. What is the reason for it to be here?